.github/workflows/ci-docs-tests.yml

```diff
@@ -0,0 +1,95 @@
+name: Docs Tests
+on:
+  push:
+    branches:
+    - main
+    tags:
+    - '*'
+  pull_request:
+    types: [opened, synchronize, labeled]
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  changes:
+    name: Check for file changes
+    runs-on: ubuntu-latest
+    outputs:
+      backend: ${{ steps.filter.outputs.backend }}
+      docker: ${{ steps.filter.outputs.docker }}
+      docs: ${{ steps.filter.outputs.docs }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: RasaHQ/pr-changed-files-filter@c4f7116a04b8a4596313469429e2ad235f59d9c4
+        id: filter
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          filters: .github/change_filters.yml
+
+  test_documentation:
+    name: Test Documentation
+    runs-on: ubuntu-latest
+    needs: [ changes ]
+
+    steps:
+    - name: Checkout git repository 🕝
+      if: needs.changes.outputs.docs == 'true'
```

Comment:
It runs only if there are changes, did you test it if everything here is ok?

Reply:
In CI workflow, [the docs tests job needs [ changes ]](https://github.com/RasaHQ/rasa/blob/c7cfac559ea37b230b0d8dfdb6e0c5a9e6c43055/.github/workflows/continous-integration.yml#L53). When I put Test Documentation to another workflow, I also copied the [`changes` job](https://github.com/RasaHQ/rasa/blob/c7cfac559ea37b230b0d8dfdb6e0c5a9e6c43055/.github/workflows/continous-integration.yml#L35-L48). maybe we don't need to do it. Instead, we can let the `wait for doc tests` needs `changes` in CI. Let me try if it works.

Reply:
Logic is ok, you can leave it as it is. What I meant here you should test if a new workflow works by changing the if condition, but only for test purposes.